config/label_sync/labels.yaml

```diff
@@ -27,39 +27,66 @@ default:
       description: Parked issue that required triaging/revisit in a near future.
       name: kind/TBD
       target: issues
+      prowPlugin: label
       addedBy: humans
     - color: 7057ff
       description: Denotes an issue ready for a new contributor.
       name: kind/good-first-issue
       target: issues
+      prowPlugin: label
+      addedBy: anyone
+    - color: e11d21
+      description: Categorizes issue or PR as related to adding, removing, or otherwise changing an API
+      name: kind/api-change
```

Comment:
how does this label relate to `area/api`?

Reply:
area/api is for issues related to api, kind/api-change means the api is changing?